adal/build.gradle

```diff
@@ -41,6 +41,13 @@ android {
         }
     }
 
+    sourceSets {
+        main {
+            java {
+                srcDir 'src/telemetry/java'
```

Comment:
no, its done by default, same with AndroidTest

---
In reply to: [76130688](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/703#discussion_r76130688) [](ancestors = 76130688)